Doc/library/unittest.mock.rst

```diff
@@ -514,6 +514,19 @@ the *new_callable* argument to :func:`patch`.
             >>> mock.call_count
             2
 
+        For :class:`AsyncMock` the :attr:`call_count` is only iterated if the function
+        has been awaited:
+
+            >>> mock = AsyncMock()
+            >>> mock()  # doctest: +SKIP
+            <coroutine object AsyncMockMixin._mock_call at ...>
+            >>> mock.call_count
+            0
+            >>> async def main():
+            >>>    await mock()
```

Comment:
```suggestion
            ...    await mock()
```

Reply:
I think we can directly do `>>> await mock()` on doctest once I somehow get to https://bugs.python.org/issue37006 :)

Reply:
Is there a way to run just the doctests? Somehow I never know how to find these errors until CI raises them.

Reply:
Yes, I think the steps in travis is good to run the tests https://travis-ci.org/python/cpython/jobs/582713783#L1482 . `xvfb-run` is required for turtle related tests. It works on my Linux machine not sure how to install `xvfb-run` on Mac.

```
$ make -C Doc/ PYTHON=../python venv
$ xvfb-run make -C Doc/ PYTHON=../python SPHINXOPTS="-q -W -j4" doctest
```